mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -1273,7 +1285,11 @@ trait Completions { this: MetalsGlobal =>
               tail
                 .map(_.edit.getNewText())
                 .mkString(
-                  s"match {\n\t${head.edit.getNewText} $$0\n\t",
+                  if (clientSupportsSnippets) {
+                    "match {\n\t${head.edit.getNewText} $$0\n\t"
+                  } else {
+                    "match {\n\t${head.edit.getNewText}\n\t"
```

Comment:
in this case we're ok with an "annoying" cursor position (`match {...}<CURSOR>`) since the exhaustive match feature is quite useful regardless (and it's opt-in anyway)